src/DataAccess/Migrations/Version20240528083322.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\AddressChangeContext\DataAccess\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240528083322 extends AbstractMigration {
+
+	public function getDescription(): string {
+		return 'Add indexes for full text search on the Address table';
+	}
+
+	public function up( Schema $schema ): void {
+		$table = $schema->getTable( 'address' );
+		$table->addIndex( [ 'first_name' ], 'idx_first_name_address_change' );
```

Comment:
This statement just adds regular indexes. For fulltext search (like we use in donations) the statement would need to be 
```suggestion
		$table->addIndex( [ 'first_name' ], 'idx_first_name_address_change', [ 'fulltext' ] );
```